.github/release.yml

```diff
@@ -0,0 +1,23 @@
+changelog:
+  exclude:
```

Comment:
I understand why that'd be important yeah.
Since this was pretty much adapted from another plugin, and the issue may also apply there, do you think it'd make sense to replicate on other packer plugins too?